Add missing meta descriptions (#1074)
mongodb/docs-node: source/usage-examples/updateMany.txt

Update Multiple Documents
=========================

.. meta::

>    :description: Update multiple documents in a MongoDB collection using the `updateMany()` method with a filter and update document.

Reviewer: ```suggestion
   :description: Update multiple documents in a collection with the MongoDB Node.js Driver by using the updateMany() method with a filter.
```
